config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/EncassArgument.java

```diff
@@ -6,10 +6,15 @@
 
 package com.ca.apim.gateway.cagatewayconfig.beans;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonPropertyOrder({"type", "name"})
 public class EncassArgument {
 
     private String name;
     private String type;
+    @JsonIgnore
```

Comment:
We need every detail of encass-argument. 
Could you please take care of populating requireExplicit (GuiPrompt) and label (GuiLabel) and reflecting back into metadata?
Please note that, ignore [label] property while preparing the bundle entity resource.

Reply:
Done!